packages/protocol/contracts/layer1/forced-inclusion/TaikoWrapper.sol

```diff
@@ -119,7 +119,7 @@ contract TaikoWrapper is EssentialContract, IProposeBatch {
         require(p.isForcedInclusion, ITaikoInbox.InvalidForcedInclusion());
 
         // Need to make sure enough transactions in the forced inclusion request are included.
```

Comment:
Think the main confusion point was the unclarity on whether forced-inclusion batch submitters have to sufficiently fill their batch to some amount. Would be good to clarify that that is not the case, like:
```suggestion
        // Preconfer must consume `MAX_TXS_PER_FORCED_INCLUSION` txs in a forced-inclusion request.
        // Note that the forced-inclusion request may contain fewer than `MAX_TXS_PER_FORCED_INCLUSION`
        // txs; the L2 execution will run the provided txs.  This rule is purely to prevent the
        // preconfer from bypassing the forced-inclusion mechanism with, e.g., `numTransactions = 0`.
```